packages/go/ein/ad.go

```diff
@@ -289,6 +289,31 @@ func ParseUserMiscData(user User) []IngestibleRelationship {
 		))
 	}
 
+	// CoerceToTGT / unconstrained delegation
+	uncondel := user.UnconstrainedDelegation
+	uncondelProps, _ := user.Properties[strings.ToLower(ad.UnconstrainedDelegation.String())].(bool) // SH v2.5.7 and earlier have unconstraineddelegation under 'Properties' only
+	if uncondel || uncondelProps {
+		domainsid := user.DomainSID
+		if domainsid == "" { // SH v2.5.7 and earlier have domainsid under 'Properties' only
+			domainsid, _ = user.Properties[strings.ToLower(ad.DomainSID.String())].(string)
+		}
+
+		data = append(data, NewIngestibleRelationship(
+			IngestibleSource{
+				Source:     user.ObjectIdentifier,
+				SourceType: ad.User,
+			},
+			IngestibleTarget{
+				Target:     domainsid,
+				TargetType: ad.Domain,
+			},
+			IngestibleRel{
+				RelProps: map[string]any{"isacl": false},
+				RelType:  ad.CoerceToTGT,
+			},
+		))
+	}
+
```

Comment:
I recommend handling nil value checks explicitly related to the map access. We have some safe property value utilities that we can reach for here.

```suggestion
	// CoerceToTGT / unconstrained delegation
	var (
		userProps        = graph.AsProperties(user.Properties)
		uncondel, _      = userProps.GetOrDefault(ad.UnconstrainedDelegation.String(), user.UnconstrainedDelegation).Bool() // SH v2.5.7 and earlier have unconstraineddelegation under 'Properties' only
		domainsid, _     = userProps.GetOrDefault(ad.DomainSID.String(), user.DomainSID).String()                           // SH v2.5.7 and earlier have domainsid under 'Properties' only
		validCoerceToTGT = uncondel && domainsid != ""
	)

	if validCoerceToTGT {
		data = append(data, NewIngestibleRelationship(
			IngestibleSource{
				Source:     user.ObjectIdentifier,
				SourceType: ad.User,
			},
			IngestibleTarget{
				Target:     domainsid,
				TargetType: ad.Domain,
			},
			IngestibleRel{
				RelProps: map[string]any{"isacl": false},
				RelType:  ad.CoerceToTGT,
			},
		))
	}

```